plasma_framework/contracts/src/exits/utils/BondSize.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.5.0;
+
+import 'openzeppelin-solidity/contracts/math/SafeMath.sol';
+
+library BondSize {
+    using SafeMath for uint256;
+
+    struct Params {
+        uint256 currentGasPrice;
+        uint256 challengeGasCost;
+        uint256 ewmaFactor;
```

Comment:
If I understand it correctly, `ewmaFactor` and `safetyFactor` would be some constant number and small. I guess we can use `uint128` for the two so they would only need one slot of storage space. Or, probably we can pack the `challengeGasCost` too, and use sth like `uint96`(?).